save-backend/src/main/kotlin/com/saveourtool/save/backend/repository/ContestRepository.kt

```diff
@@ -72,4 +77,20 @@ JpaSpecificationExecutor<Contest> {
      * @return [Page] of [Contest]s that belong to organization with name [organizationName]
      */
     fun findByOrganizationName(organizationName: String, pageable: Pageable): Page<Contest>
+
+    fun findByStatusNotAndEndTimeAfterOrderByCreationTimeDesc(status: ContestStatus, currentTime: LocalDateTime, pageable: Pageable): List<Contest>
+
+    @Query("""select contest_id from save_cloud.featured_contests""", nativeQuery = true)
```

Comment:
very simple query, why native and not JPA?

Reply:
As far as I understand, we are required to have an entity if we want to use JPQL while there is no need in entity for such a simple table